sdks/python/apache_beam/runners/worker/operations.py

```diff
@@ -286,6 +287,10 @@ def _read_side_inputs(self, tags_and_types):
     # provided directly.
     assert self.side_input_maps is None
 
+    # Get experiments active in the worker to check for side input metrics exp.
+    experiments = set(RuntimeValueProvider(
```

Comment:
Should we add RuntimeValueProvider.get(option_name) as a class method to avoid creating dummy instance like this?